napari_builtins/io/_read.py

```diff
@@ -133,6 +133,12 @@ def read_zarr_dataset(path: str):
         ]
         assert image, 'No arrays found in zarr group'
         shape = image[0].shape
+    elif (path / 'zarr.json').exists():
+        # zarr v3
+        import zarr
+
+        image = zarr.open(store=path)
+        shape = image.shape
```

Comment:
@Czaki this use case probably also needs to be split into single array/group of arrays? I presume that we can make the earlier ifs more complex: "if there's a .zarray *or* there's a zarr.json and it only contains one array"

Reply:
I'm not familiar with zarr arrays. I just follow the tests. And now waiting on new Zarr release.